src/theory/quantifiers/sygus/synth_conjecture.cpp

```diff
@@ -1081,13 +1082,33 @@ void SynthConjecture::printSynthSolution(std::ostream& out)
       if (is_unique_term)
       {
         out << "(define-fun " << f << " ";
-        if (dt.getSygusVarList().isNull())
+        // Only include variables that are truly bound variables of the
+        // function-to-synthesize. This means we exclude variables that encode
+        // external terms. This ensures that --sygus-stream prints
+        // solutions with no arguments on the predicate for responses to
+        // the get-abduct command.
+        std::vector<Node> pvs;
```

Comment:
What is the job of pvs?

Reply:
These are the variables that will be printed in the argument list.